molgenis-jobs/src/main/java/org/molgenis/data/jobs/JobBootstrapper.java

```diff
@@ -36,7 +36,7 @@ public void bootstrap()
 
 	private void bootstrap(SystemEntityMetaData systemEntityMetaData)
 	{
-		dataService.query(systemEntityMetaData.getName()).eq(STATUS, RUNNING).findAll().forEach(this::setFailed);
+		dataService.query(systemEntityMetaData.getName()).eq(STATUS, RUNNING).or().eq(STATUS, PENDING).findAll().forEach(this::setFailed);
```

Comment:
After discussion at standup today, I apparently was mistaken, and pending jobs are "picked up" and executed after the restart. So this line can be removed again.